unable to install mockgen following the instructions in README

I'm unable to install `mockgen` following the instructions in https://github.com/golang/mock#installation:

```
$ go get -v github.com/golang/mock/gomock
github.com/golang/mock (download)
created GOPATH=/home/dmitris/go; see 'go help gopath'
github.com/golang/mock/gomock
$ go install -v github.com/golang/mock/mockgen
go/src/github.com/golang/mock/mockgen/parse.go:34:2: cannot find package "golang.org/x/tools/go/packages" in any of:
	/usr/local/go/src/golang.org/x/tools/go/packages (from $GOROOT)
	/home/dmitris/go/src/golang.org/x/tools/go/packages (from $GOPATH)
```

To make `go install` work, I had to also run:
`go get -v golang.org/x/tools/go/packages`.  Alternatively, the following command works:
`go get -v -u github.com/golang/mock/mockgen` - maybe we could use it instead of the current pair of `go get` and `go install`?

`go version` output: `go version go1.12.6 linux/amd64`

--- a/README.md
+++ b/README.md
@@ -9,12 +9,12 @@ contexts too.
 Installation
 ------------
 
-Once you have [installed Go][golang-install], run these commands
-to install the `gomock` package and the `mockgen` tool:
+Once you have [installed Go][golang-install], install the `mockgen` tool:
 
-    go get github.com/golang/mock/gomock
-    go install github.com/golang/mock/mockgen
+    go get github.com/golang/mock/mockgen
 
+_Note: It is recommended to have `GO111MODULE=on` to ensure the correct
+dependencies are used._
 
 Documentation
 -------------
@@ -74,9 +74,9 @@ It supports the following flags:
 
 *  `-build_flags`: (reflect mode only) Flags passed verbatim to `go build`.
 
-* `-mock_names`: A list of custom names for generated mocks. This is specified 
+* `-mock_names`: A list of custom names for generated mocks. This is specified
 	as a comma-separated list of elements of the form
-	`Repository=MockSensorRepository,Endpoint=MockSensorEndpoint`, where 
+	`Repository=MockSensorRepository,Endpoint=MockSensorEndpoint`, where
 	`Repository` is the interface name and `MockSensorRepository` is the desired
 	mock name (mock factory method and mock recorder will be named after the mock).
 	If one of the interfaces has no custom name specified, then default naming

